File: go/1038.go
package main

import "fmt"

func main() {
	var product, quantity int
	fmt.Scanf("%d %d", &product, &quantity)
	price := []float64{4, 4.5, 5, 2, 1.5}
	fmt.Printf("Total: R$ %.2f\n", float64(quantity)*price[product-1])
}

File: C/src/1038.c
#include <stdio.h>

int main() {
  int quantity, product;
  float price[5] = {4, 4.5, 5, 2, 1.5};

  scanf("%i %i", &product, &quantity);
  printf("Total: R$ %.2f\n", (quantity * price[product - 1]));
  return 0;
}